module_utils/network_lsr/argument_validator.py

```diff
@@ -2012,3 +2084,106 @@ def _ipv6_is_not_configured(connection):
                             "match.path is not supported by the running version of "
                             "NetworkManger.",
                         )
+        self.validate_route_tables(connection, idx)
+
+
+class IPRouteUtils(object):
+
+    ROUTE_TABLE_ALIAS_RE = re.compile("^[a-zA-Z0-9_.-]+$")
+
+    @classmethod
+    def _parse_route_tables_mapping(cls, file_content, mapping):
+        for line in file_content.split(b"\n"):
+
+            #
+            # We are a bit more forgiving, but the aliases must be nice UTF-8 names.
+
+            # iproute2 only skips over leading ' ' and '\t'. We are more forgiving,
+            # and strip all leading+trailing whitespace.
+            line = line.strip()
+
+            # skip empty lines or comments
+            if not line or line[0] == ord(b"#"):
+                continue
+
+            # Split at the first space. We accept here any whitespace (unlike iproute2).
+            # Also, this will strip any leading/trailing whitespaces (unlike iproute2)
+            ll = line.split(None, 1)
+            if len(ll) != 2:
+                continue
+            line1 = ll[0]
+            line2 = ll[1]
+
+            i = line2.find(b" ")
+            if i == -1:
+                i = line2.find(b"\t")
+            if i >= 0:
+                # again, iproute2 only accepts one ' ', followed by '#'. We accept any number of
+                # space and any kind of whitespace (the split() above already partly took care
+                # of those).
+                line3 = line2[i + 1 :].strip()
+
+                # In Python 2.x, there is no new type called `bytes`. And Python 2.x
+                # `bytes` is just an alias to the str type
+                if sys.version[:1] == "2":
```

Comment:
there is `Util.PY3`

Reply:
(the advantage of using that, is that you can grep for that thing and fine all places where we need different code depending on the python version... I guess, you still could grep for sys.version too, but this seems more explicit.